lib/models/model.js

```diff
@@ -18,13 +18,68 @@ class Model {
     else return this.schema.find({});
   }
 
-  create(record) {}
 
-  update(_id, record) {}
+  async create(record) {
+    if (sanitize (record)){
+        let res = await fetch(this.url, {
+          method: 'POST',
+          body: JSON.stringify(record),
+          headers: { 'Content-Type': 'application/json' }
+        });
 
-  delete(_id) {}
+        if (res.ok) {
+          let json = await res.json();
+          console.log('Data:', json);
+        } else console.log(res.statusText);
+      }else { 
+        console.log('wrong format!');
+      } 
+  }
```

Comment:
I'm not sure what this section of code is doing; you're using a client request in your model, when really you should just be calling Mongoose functions from your schema. This is not the correct content for `model.js`

Reply:
Since you're creating a record in MongoDB, you should just be calling `this.schema.save(record)` here